apps/coscrad-frontend/src/components/commands/connections/create-note-form.tsx

```diff
@@ -0,0 +1,69 @@
+import { LanguageCode } from '@coscrad/api-interfaces';
+import { isNonEmptyString } from '@coscrad/validation-constraints';
+import { FormControl, FormGroup, MenuItem, Select, TextField, Typography } from '@mui/material';
+import { useState } from 'react';
+
+export const CreateNoteForm = (): JSX.Element => {
+    const [text, setText] = useState<string>('');
+
+    const [languageCode, setLanguageCode] = useState<LanguageCode>(null);
+
+    const isFormComplete =
+        isNonEmptyString(text) && Object.values(LanguageCode).includes(languageCode);
+
+    return (
+        <div>
```

Comment:
Use `Box` or `<></>` fragment.  Fragment doesn't clutter the HTML when we're trying to troubleshoot what styles MUI is adding automatically.

Reply:
done